addons/mail/models/mail_activity.py

```diff
@@ -504,6 +504,25 @@ def action_feedback_schedule_next(self, feedback=False):
             'target': 'new',
         }
 
+    def _prepare_next_activity_values(self):
+        """ Prepare the next activity values based on the current activity record and applies _onchange methods
+        :returns a dict of values for the new activity
+        """
+        self.ensure_one()
+        vals = self.default_get(self.fields_get())
+
+        vals.update({
+            'previous_activity_type_id': self.activity_type_id.id,
+            'res_id': self.res_id,
+            'res_model': self.res_model,
+            'res_model_id': self.env['ir.model']._get(self.res_model).id,
+            'summary': self.summary,
```

Comment:
DO not hide changes in a move ... Why this change ? Why should we keep summary from a given activity to the next one ? Chances are high this will be unrelated to the new activity.

Reply:
It was not my intention to "hide" the change.
Indeed you are right, there is a good chance the summary should be a new one, depending on the next activity - however, the `_onchange_activity_type_id()` will set the summary according to the activity type - and it is called after this line, will therefore overwrite the `summary` value.
This will ONLY apply in the event that there is no summary defined in the activity type.

The reason for this is so that we can have a document request that is named appropriately. Without this change, we have the new document request that is named according to the previous document uploaded. This way, it is named according to the previous document request.

Would you prefer a separate commit for this?

Reply:
Or alternatively, it could be done in the documents override - and ensure it is only done for document related activities

Reply:
removed - I manage this in the documents override instead